### PR TITLE
test(image): Co-locate images

### PR DESCRIPTION
**Pull request by obartra**
Use `storyFolders` to get the image names and colocate test images. This should make updating them easier

### CODE REVIEW

File: storybook/shared/storyFolders.js [hunk 1]
```diff
@@ -28,6 +28,24 @@ function getFilesAndFolders(
...
 }
 
+function getImage(files, filepath) {
+  const pngFile = filepath.replace(/\.js$/, '.spec.png')
+  const hasPNG = files.indexOf(pngFile) !== -1
+
+  return hasPNG ? pngFile : ''
+}
+
+function dropEnds(array) {
```
Comment (arahansen) [inline storybook/shared/storyFolders.js]: `const dropEnds = flow(tail, initial)`

Comment (obartra) [inline storybook/shared/storyFolders.js]: yeah I rather avoid importing `flow` for this one case

Comment (arahansen) [inline storybook/shared/storyFolders.js]: Are we concerned with bundle size for a dev tool?

Comment (obartra) [inline storybook/shared/storyFolders.js]: oops good point we are not, I totally missed that, will update

**Pull request merged**